README.md

```diff
@@ -152,9 +152,13 @@ If the script is connecting to any local apps/services you need to forward the p
 container. The easiest way to do this is to use `--net=host` argument for `docker run`:
 
 ```
-docker run --net=host digitalmarketplace/scripts scripts/index-services.py dev ...
+docker run --net=host digitalmarketplace/scripts scripts/index-to-search-service.py dev ...
 ```
 
+**Note that this will use the latest image built from master, and therefore (at present) all
+merges to master have effect almost immediately in Preview, Staging and Production environments.**
+This is discussed at <https://trello.com/c/di223DXI/241-no-control-of-scripts-repo-on-jenkins>.
```

Comment:
Linking to a Trello ticket from this readme seems super-weird.  And I'm not at all convinced that this is tech debt - I'd totally expect scripts to be run once they're merged to master.